anaconda.spec.in

```diff
@@ -34,7 +34,7 @@ Source0: %{name}-%{version}.tar.bz2
 %define mehver 0.23-1
 %define nmver 1.0.0-6.git20150107
 %define partedver 1.8.1
-%define pykickstartver 1.99.66.19
+%define pykickstartver 1.99.66.19-5
```

Comment:
Looks like pykickstart 1.99.66.20-1 got released for RHEL7 in the end, not the expected -5 patch build.

Reply:
Fixed. Thanks!